wazo_confgend/plugins/pjsip_conf.py

```diff
@@ -841,10 +820,13 @@ def __init__(self, dependencies):
         self._config_file_generator = AsteriskConfFileGenerator()
 
     def generate(self):
+        asterisk_file_generator = AsteriskFileGenerator(asterisk_file_dao)
+        output = StringIO()
+        asterisk_file_generator.generate('pjsip.conf', output)
```

Comment:
Should we raise or stop the generation if there are no global section? If there are no global section, then no error/warning is raised and the global section is *just* missing.

Reply:
> Should we raise or stop the generation if there are no global section? If there are no global section, then no error/warning is raised and the global section is _just_ missing.

That's a good question. In theory having no global section and using the defaults seems to be valid, I'll check if the practice says otherwise...

After checking, the module is happy without a global section but our other configuration override the global section using `[global](+)` in the `05-autoprov-wizard.conf` file. That emits a warning and I think we're better with less warnings.

I think we should generate an empty `global` section and log an error in the confgend log if the file is missing from the DB. I would not prevent the config from being generated.